src/components/interface/Callout/Callout.js

```diff
@@ -12,13 +12,20 @@ import '@gouvfr/dsfr/dist/component/callout/callout.css';
  * @visibleName Callout
  */
 const Callout = ({
-  hasInfoIcon, children, className, colorFamily, colors, ...remainingProps
+  hasInfoIcon, icon, children, className, colorFamily, colors, ...remainingProps
 }) => {
+  if (!hasInfoIcon) { // TODO Delete deprecated prop hasInfoIcon when reaching major version 4.0.0
```

Comment:
can you put the @deprecated in the ts part please ?

Reply:
No problem, I'll add it